paddle/function/GemmConvOp.cpp

```diff
@@ -233,6 +233,11 @@ class GemmConvGradInputFunction : public ConvFunctionBase {
       inputGrad += inputChannels * inputHeight * inputWidth;
       outputGrad += outputChannels * outputHeight * outputWidth;
     }
+#ifdef PADDLE_MOBILE_INFERENCE
```

Comment:
Should fix `GemmConvFunction` not `GemmConvGradInputFunction`.

Reply:
Oh... I'm sorry. I'll fix it  :)

Reply:
Done.